h2/src/test/org/h2/samples/SQLServerSample.java

```diff
@@ -0,0 +1,48 @@
+package org.h2.samples;
+
+import org.h2.tools.DeleteDbFiles;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class SQLServerSample {
+
+    /**
+     * Called when ran from the command line.
+     *
+     * @param args ignored
+     */
+    public static void main(String... args) throws Exception {
+        Class.forName("org.h2.Driver");
+        Connection conn = DriverManager.getConnection("jdbc:h2:mem:;MODE=MSSQLServer");
+        Statement stat = conn.createStatement();
+
+        stat.execute("create table parent(id int primary key identity, name varchar(255))");
+        stat.execute("create table child(id int primary key identity, name varchar(255), parent_id int, foreign key (parent_id) references public.parent(id) )");
```

Comment:
This line is too long, only 120 characters for code and 80 for comments are allowed in H2.